getblocklab/block-lab
# Bump BL version to 1.5.6

#### Changes
* Bumps BL to `1.5.6`
* Uses Nick's fix of the `isLarge` prop warning
* Doesn't include any of the migration to GCB, that's in a separate branch
* Updates almost all `npm` packages
* No new feature
* Heres [the diff](https://github.com/getblocklab/block-lab/compare/v1.5.5...bump/1.5.6) since the last release. A lot of it is new JS component and e2e tests, which of course aren't deployed to wp.org.


#### Testing instructions
1. `rm js/editor.blocks.asset.php js/editor.blocks.js css/blocks.editor.css` # remove built files, to ensure this is using the most recently built versions, not falling back to previous builds
2. `npm install && npm run dev`
3. Regression testing of plugin, especially in the block editor. It looked good in my testing.
